extensions/amp-accordion/1.0/base-element.js

```diff
@@ -255,6 +255,10 @@ BaseElement['detached'] = true;
 
 /** @override */
 BaseElement['props'] = {
-  'animate': {attr: 'animate', type: 'boolean'},
-  'expandSingleSection': {attr: 'expand-single-section', type: 'boolean'},
+  'animate': {attr: 'animate', type: 'boolean', media: true},
+  'expandSingleSection': {
+    attr: 'expand-single-section',
```

Comment:
Slightly in doubt about this one. I'd look at it as media reserved for very "UI" concepts. And this one feels a bit more structural.